app/src/main/res/layout/activity_login.xml

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="utf-8"?>
+<android.support.constraint.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".LoginActivity"
+    android:paddingStart="@dimen/activity_horizontal_padding"
+    android:paddingEnd="@dimen/activity_vertical_padding"
+    android:paddingTop="@dimen/activity_horizontal_padding"
+    android:paddingBottom="@dimen/activity_vertical_padding">
+
+    <android.support.design.widget.TextInputEditText
+        android:id="@+id/et_login_username"
+        android:layout_width="300dp"
```

Comment:
Almoooooooost!!  You started great!!! Please change this hardcoded values to a variable in the dimens resources